### ISSUE TITLE
Add Effector to similar package

### ISSUE THREAD

**Issue by PowerSlime** [pastelsky/bundlephobia#154]
## Type
Add [Effector](https://www.npmjs.com/package/effector) to similar package of [Redux](https://www.npmjs.com/package/redux)

## Package name
Effector

**Comment by pastelsky** [comment]
Thanks for the suggestion. However, the state-management category is pretty busy and we want to offer well-known, maintained, and relevant libraries as focussed suggestions in Bundlephobia. This usually means we keep suggestions in a category down to 4-5.

**Comment by sergeysova** [comment]
@pastelsky But effector is well-known and maintained state manager, that relevant to this category

**Comment by mg901** [comment]
[codersrank](codersrank.io) also recognizes effector
<img width="892" alt="Снимок экрана 2020-05-20 в 14 48 55" src="https://user-images.githubusercontent.com/7874664/82442915-3ce4ce80-9aa9-11ea-90a3-62cb80c91c49.png">

**Comment by Natteke** [comment]
Finally some1 made an issue...

+1

**Comment by zerobias** [comment]
This was closed wrong, we waited over a year for approval, just to see the library finally get silently thrown out [just before release](https://github.com/pastelsky/similar-npm-packages/pull/7/commits/ace1ba8240d6827c712e100736c5237d8e8d56a1) and the discussion being closed without any single word of conversation, it's disappointing.

"Way more popular" cannot have 20k/month downloads which is literally nothing compared with millions mobx/redux, so what’s the key of replacement? Are we need bundlephobia label in readme?
